src/Microsoft.OData.Edm/ExtensionMethods/EnumHelper.cs

```diff
@@ -126,6 +126,48 @@ public static bool TryParseEnum(this IEdmEnumType enumType, string value, bool i
             return true;
         }
 
+        /// <summary>
+        /// Parse an enum integral value to enum member.
+        /// </summary>
+        /// <param name="enumType">edm enum type</param>
+        /// <param name="value">input integral value.</param>
+        /// <param name="enumMember">parsed result.</param>
+        /// <returns>true if parse succeeds, false if parse fails.</returns>
+        public static bool TryParseEnum(this IEdmEnumType enumType, long value, out IEdmEnumMember enumMember)
+        {
+            enumMember = null;
+            foreach (IEdmEnumMember member in enumType.Members)
+            {
+                if (member.Value.Value == value)
+                {
+                    enumMember = member;
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        /// <summary>
+        /// Checks if the given member name exists in the enum type.
+        /// </summary>
+        /// <param name="enumType">The enum type.</param>
+        /// <param name="memberName">The member name to check.</param>
+        /// <param name="comparison">The comparison type to use for string comparison. Default is Ordinal.</param>
+        /// <returns>True if the member name exists in the enum type; otherwise, false.</returns>
+        public static bool EnumMemberExists(this IEdmEnumType enumType, string memberName, StringComparison comparison = StringComparison.Ordinal)

```

Comment:
I feel that `ContainsMember` is probably a better fit since you're checking if the member specified as a string is contained in the enum - `myEnum.ContainsMember("foobar")`

Reply:
@gathogojr 

Yes, this is correct. I have renamed it to `ContainsMember`